R/wrangle_and_check.R

```diff
@@ -178,11 +178,13 @@ check_financial_data <- function(financial_data, asset_type) {
 #' Check company term values for plausibility
 #'
 #' @param data A tibble holding company_terms data.
+#' @param interactive_mode If TRUE the, more verbose, interactive mode is used.
 #'
 #' @return Returns data invisibly.
-check_company_terms <- function(data) {
+#' @export
+check_company_terms <- function(data, interactive_mode = FALSE) {
   not_na_terms <- data %>%
-    dplyr::filter(!is.na(term)) %>%
+    dplyr::filter(!is.na(.data$term)) %>%
     dplyr::pull(term)
```

Comment:
how come you only add `.data$` masking in the `filter` part of the pipe, but not in `pull()`?

Reply:
ignorance, changing it